test/built-ins/Promise/race/name.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2021 Sergey Rubanov. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 25.4.4.3
+description: Promise.race `name` property
+info: |
+    ES6 Section 17:
+
+    Every built-in Function object, including constructors, that is not
+    identified as an anonymous function has a name property whose value is a
+    String. Unless otherwise specified, this value is the name that is given to
+    the function in this specification.
+
+    [...]
+
+    Unless otherwise specified, the name property of a built-in Function
+    object, if it exists, has the attributes { [[Writable]]: false,
+    [[Enumerable]]: false, [[Configurable]]: true }.
+includes: [propertyHelper.js]
+---*/
+
+assert.sameValue(Promise.race.name, 'race');
```

Comment:
the value should be `allSettled`, right? Ref https://github.com/tc39/ecma262/pull/2226#discussion_r520910084

Reply:
The comment you point to suggests using `anySettled`, not `allSettled`.

But this is something I will bring up in committee. I think the safest/most conservative option is to preserve the pre-existing function object, i.e. have the name be `race`. This matches what happened with `trimLeft`/`trimRight`.

It would be nice (IMHO, nicer) to have the name be `anySettled`, but there is a minor web compat risk, since it would change `Promise.race.name` to `'anySettled'`.

Reply:
I 💯 agree

Reply:
> The comment you point to suggests using `anySettled`, not `allSettled`.

It was a typo. Thanks!

Reply:
I'm not taking any strong sides but I have a preference for the name being `anySettled`. I understand it's a breaking change but it feels like one we can go through without problems.